site/assets/scss/_syntax.scss

```diff
@@ -1,108 +1,126 @@
-// stylelint-disable comment-empty-line-before, declaration-block-single-line-max-declarations
+// How to retrieve the list of all Chroma classes
+//    $ hugo gen chromastyles > chroma.css
+//    $ cat chroma.css
 
-/* Background .chroma { background-color: $white; } */
-/* Other .chroma .x { } */
-/* Error */ .chroma .err { color: $red-500; }
-/* LineTableTD .chroma .lntd { vertical-align: top; padding: 0; margin: 0; border: 0; } */
-/* LineTable .chroma .lntable { border-spacing: 0; padding: 0; margin: 0; border: 0; width: auto; overflow: auto; display: block; } */
-/* LineHighlight .chroma .hl { display: block; width: 100%; background-color: #ffffcc; } */
-/* LineNumbersTable .chroma .lnt { margin-right: .4em; padding: 0 .4em; } */
-/* LineNumbers .chroma .ln { margin-right: .4em; padding: 0 .4em; } */
+// Note: Bootstrap stops at --base0F
 
-/* Comment */ .chroma .c { color: $gray-700; }
-/* CommentHashbang */ .chroma .ch { font-style: italic; color: $cyan-700; }
-/* CommentMultiline */ .chroma .cm { color: $gray-700; }
-/* CommentPreproc */ .chroma .cp { color: $cyan-800; }
-/* CommentPreprocFile */ .chroma .cpf { color: $green-700; }
```

Comment:
Seems to be missing

Reply:
It's missing in BS as well. Let's let it as it is since it is not used in the documentation (not found with `find _site -name "*.html" |xargs grep "\"cpf\""`)